src/main/java/com/zaxxer/hikari/pool/HikariPool.java

```diff
@@ -580,6 +584,7 @@ private void checkFailFast()
     */
    private void throwPoolInitializationException(Throwable t)
    {
+      logger.error("{} - Exception during pool initialization.", poolName, t);
```

Comment:
I merged a pull request removing this line just an hour ago. Was re-adding it a mistake?

Reply:
Hello Leo,
That was certainly not intended. Sorry for this!
I have fixed it and pushed it to my fork.
Kind regards,
Valentin

Reply:
Hi @lfbayer ,
As mentioned I while ago this has been updated.
Can you please re-verify it and merge it if possible?

Thanks!
Valentin